frontend/packages/kubevirt-plugin/src/components/vm-disks/vm-disks.tsx

```diff
@@ -47,6 +61,7 @@ const getStoragesData = ({
       diskInterface: disk.getDiskInterface(),
       size: disk.getReadableSize(),
       storageClass: disk.getStorageClassName(),
+      pvc: disk.getPVCName() || lookupPVCName(pvcs, disk),
```

Comment:
I agree that the method getPVCName is a bit misleading. Can we rename that to getPVCNameBySource and create a new true method for `getPVCName` which would makes things easier for use cases like this one?

Btw, getPVCNameBySource is used to determine the pvc just for the StorageUISource types we support - for example DiskModal. So it is not sufficient for even other unsuported source like the DiskRow

Reply:
done